forms/json-output/metadata/config/config.meta.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
While there is nothing to stop us sharing forms across runner apps -- as we were already doing -- I've switched to the pattern I've been using in `fb-component-acceptance-tests` where forms and runners aren't shared